skimage/feature/corner.py

```diff
@@ -394,15 +388,20 @@ def _symmetric_compute_eigenvalues(S_elems):
         ith-largest eigenvalue at position (j, k).
     """
 
-    if len(S_elems) == 3:  # Use fast Cython code for 2D
-        eigs = np.stack(_image_orthogonal_matrix22_eigvals(*S_elems))
+    if len(S_elems) == 3:  # Fast explicit formulas for 2D.
+        M00, M01, M11 = S_elems
+        eigs = np.empty((2, *M00.shape), M00.dtype)
+        eigs[:] = (M00 + M11) / 2
```

Comment:
Wouldn't it be easier and faster here to just do

```suggestion
        eigs = (M00 + M11) / 2
```
?

In this case, you are creating the temporary array and assigning the variable `eigs` to it, whereas in your current implementation, you are creating the temporary array and then copying all the values over?

Reply:
No, `(M00+M01)/2` has shape `(ni, nj)` whereas `eigs` has shape `(2, ni, nj)` but I'm filling it via broadcasting.  This is actually quite faster than explicitly stacking the array twice with np.tile or np.stack:
```
In [24]: t = np.random.rand(128, 128)

In [25]: def fasttile(t):
    ...:     u = np.empty((2, *t.shape))
    ...:     u[:] = t
    ...:     return u
    ...:     

In [26]: %timeit fasttile(t)
14.6 µs ± 6.43 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [27]: %timeit np.tile(t, (2, 1, 1))
18.6 µs ± 124 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [28]: %timeit np.stack([t, t])
20 µs ± 41.1 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)
```